core/src/OpenMP/Kokkos_OpenMP_Instance.hpp

```diff
@@ -69,11 +69,11 @@ namespace Impl {
 
 class OpenMPInternal;
 
-extern int g_openmp_hardware_max_threads;
+inline int g_openmp_hardware_max_threads = 1;
 
-extern thread_local int t_openmp_hardware_id;
+inline thread_local int t_openmp_hardware_id = 0;
 // FIXME_OPENMP we can remove this after we remove partition_master
-extern thread_local OpenMPInternal* t_openmp_instance;
+inline thread_local OpenMPInternal* t_openmp_instance = nullptr;
```

Comment:
how does this work now? Doesn't each translation unit now get its own version of this??

Reply:
Every object file has the variable in an initialized data section, and the linker folds them all to one instance

Reply:
If you know old Fortran, think `COMMON` block